### PR TITLE
Support node 16

### PR DESCRIPTION
**Pull request by gkhunger-sf**

### CODE REVIEW

**Comment by gkhunger-sf** [comment]
image

File: .circleci/config.yml [hunk 1]
```diff
@@ -29,16 +29,19 @@ executors:
   node-10:
     <<: *defaults
     docker:
-      - image: circleci/node:10.20.1
+      - image: cimg/node:10.20.1
```
Comment (wjhsf) [inline .circleci/config.yml:32]: Why the change from `circleci` to `cimg`?

File: README.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ Visit the [Commerce Cloud Developer Center](https://developer.commercecloud.com/
 Download and install Node.js and npm [here](https://nodejs.org/en/download/).
 ​
 
-> **Note:** Only Node.js version 10 and 12 LTS are supported. Other versions can cause unexpected results. To use a different version of Node.js for other projects, you can manage multiple versions of Node.js with [nvm](https://github.com/nvm-sh/nvm).
+> **Note:** Only Node.js version 10, 12, 14 and 16 LTS are supported. Other versions can cause unexpected results. To use a different version of Node.js for other projects, you can manage multiple versions of Node.js with [nvm](https://github.com/nvm-sh/nvm).
```
Comment (wjhsf) [inline README.md:15]: * We should drop official support for v10, since it's EOL. We can keep it in CI to support it unofficially.
* We don't necessarily need to start officially supporting v16 immediately. Like we did for v14, we can add it to CI and support it unofficially until it's Active LTS.

File: .circleci/config.yml [hunk 3]
```diff
@@ -29,16 +29,19 @@ executors:
...
   node-12:
     <<: *defaults
     docker:
-      - image: circleci/node:12.16.3
+      - image: cimg/node:12.16.3
```
Comment (wjhsf) [inline .circleci/config.yml:36]:
```suggestion
      - image: cimg/node:12.21.1
```

File: .circleci/config.yml (hunk 1 above)
Comment (wjhsf) [inline .circleci/config.yml:32]:
```suggestion
      - image: cimg/node:10.24.1
```

File: .circleci/config.yml [hunk 4]
```diff
@@ -29,16 +29,19 @@ executors:
...
   node-14:
     <<: *defaults
     docker:
-      - image: circleci/node:14.2.0
-
+      - image: cimg/node:14.2.0
```
Comment (wjhsf) [inline .circleci/config.yml:40]:
```suggestion
      - image: cimg/node:14.17.0
```

**Review by wjhsf — APPROVED** [review]
Since we're poking this file, maybe also upgrade each major to the latest minor.

File: .circleci/config.yml (hunk 1 above)
Comment (gkhunger-sf) [reply .circleci/config.yml:32]: image

**Review by jbegleiter-sf — APPROVED** [review]

**Review by wjhsf — APPROVED** [review]
